inst/metadata/metadata.csv

```diff
@@ -1,28 +1,28 @@
 data_type,plot_type,indicator,value_column,error_low_column,error_high_column,indicator_column,indicator_value,name,scale,accuracy,format
-survey,choropleth,prevalence,estimate,,,indicator,prevalence,t_(HIV_PREVALENCE),1,,0.00%
-survey,choropleth,art_coverage,estimate,,,indicator,art_coverage,t_(ART_COVERAGE),1,,0.00%
-programme,choropleth,art_current,art_current,,,,,t_(ART_NUMBER_ATTENDING),1,,0
-survey,choropleth,recent_infected,estimate,,,indicator,recent_infected,t_(PROPORTION_RECENTLY_INFECTED),100,100,0
+survey,choropleth,prevalence,estimate,,,indicator,prevalence,t_(HIV_PREVALENCE),1,,0.0%
+survey,choropleth,art_coverage,estimate,,,indicator,art_coverage,t_(ART_COVERAGE),1,,0.0%
+programme,choropleth,art_current,art_current,,,,,t_(ART_NUMBER_ATTENDING),1,100,0
+survey,choropleth,recent_infected,estimate,,,indicator,recent_infected,t_(PROPORTION_RECENTLY_INFECTED),1,1,0.00%
 survey,choropleth,viral_suppression_plhiv,estimate,,,indicator,viral_suppression_plhiv,t_(VLS),1,,0.00%
-anc,choropleth,anc_prevalence,anc_prevalence,,,,,t_(ANC_HIV_PREVALENCE),1,,0.00%
-anc,choropleth,anc_art_coverage,anc_art_coverage,,,,,t_(ANC_PRIOR_ART_COVERAGE),1,,0.00%
-output,choropleth,prevalence,mean,lower,upper,indicator,prevalence,t_(HIV_PREVALENCE),1,,0.00%
-output,choropleth,art_coverage,mean,lower,upper,indicator,art_coverage,t_(ART_COVERAGE),1,,0.00%
-output,choropleth,art_current,mean,lower,upper,indicator,art_current_residents,t_(ART_NUMBER_RESIDENTS),1,,0
-output,choropleth,receiving_art,mean,lower,upper,indicator,art_current,t_(ART_NUMBER_ATTENDING),1,,0
+anc,choropleth,anc_prevalence,anc_prevalence,,,,,t_(ANC_HIV_PREVALENCE),1,,0.0%
+anc,choropleth,anc_art_coverage,anc_art_coverage,,,,,t_(ANC_PRIOR_ART_COVERAGE),1,,0.0%
+output,choropleth,prevalence,mean,lower,upper,indicator,prevalence,t_(HIV_PREVALENCE),1,,0.0%
+output,choropleth,art_coverage,mean,lower,upper,indicator,art_coverage,t_(ART_COVERAGE),1,,0.0%
+output,choropleth,art_current_residents,mean,lower,upper,indicator,art_current_residents,t_(ART_NUMBER_RESIDENTS),1,100,0
+output,choropleth,art_current,mean,lower,upper,indicator,art_current,t_(ART_NUMBER_ATTENDING),1,100,0

```

Comment:
art_current is no receiving ART within district? And residents is number received ART within district who are resident in the district?